src/main/kotlin/blackjack/model/Players.kt

```diff
@@ -2,26 +2,26 @@ package blackjack.model
 
 class Players(val values: List<Player>) {
     fun findNotOver(): List<Player> {
-        return values.filter { !it.gameOver }
+        return values.filter { !it.stay }
     }
 
-    fun setGameOver(player: Player): Players {
+    fun stay(player: Player): Players {
         return Players(
             values.map {
                 if (player.name.equals(it.name)) {
```

Comment:
```suggestion
                if (player.name == it.name) {
```
이제보니 equals를 사용하는 곳들에서 `==` 키워드로 대체할 수 있겠군요! 다음 문서도 참고해보시면 좋습니다.

https://kotlinlang.org/docs/equality.html